orc8r/cloud/go/services/orchestrator/obsidian/models/conversion.go

```diff
@@ -626,6 +627,21 @@ func (m *GatewayVpnConfigs) FromBackendModels(ctx context.Context, networkID str
 	return nil
 }
 
+func LastGatewayCheckInWasRecent(gatewayStatus *GatewayStatus, magmadConfig *MagmadGatewayConfigs) bool {
+	if gatewayStatus == nil {
+		return false
+	}
+	checkinTime := time.UnixMilli(int64(gatewayStatus.CheckinTime))
+
+	var checkinInterval time.Duration
+	if magmadConfig != nil {
+		checkinInterval = time.Duration(int64(magmadConfig.CheckinInterval)) * time.Second
+	} else {
+		checkinInterval = time.Minute
+	}
+	return time.Now().Before(checkinTime.Add(5 * checkinInterval))
```

Comment:
The grace period of 5 intervals is somewhat arbitrary. It is based on existing behavior (grace period of 5 minutes for a default interval of 1 minute).

Reply:
Same here, lets create a constant and set it up on top. 

Add a comment on each constant to have some kind of idea what they mean

Reply:
Done.